healpix_alchemy/tests/test_types.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+from mocpy import MOC
+import itertools
+from astropy_healpix import HEALPix
+
+from .. import types
+
+
+def random_MOC(max_l):
+    json = {str(level): list(
+        np.unique(
+            np.random.randint(0, 12 * (4 ** level),
+                              np.random.randint(12 * (4 ** level)))))
+            for level in range(max_l)}
```

Comment:
Use the new-style numpy Generator API rather than the legacy global random state (see https://numpy.org/doc/stable/reference/random/index.html). Also, note that there is already a fixture in the benchmarks folder to generate a randomly subdivided sky map.

Rather than generate a random MOC, why not just generate a MOC for some simple known polygon or cone?

Reply:
I thought generating a random one is again more "complete" since it can cover wacky shapes or whatever, but I am again happy to just randomly select a cone of healpix pixels since the random selection is then just pick a center point and radius, which is much faster & easier